lecture-gen.py

```diff
@@ -27,38 +31,112 @@ def validate_start_date(date):
 
 def add_weeks(start_datetime,weeks):
     """
-    Return a datetime that is start_datetime plus weeks into the future 
+    Return a datetime that is start_datetime plus weeks into the future
 
-    Example: 
+    Example:
       start_datetime represents 2019-03-31, weeks is 0 => 2019-03-31
       start_datetime represents 2019-03-31, weeks is 1 => 2019-04-07
       start_datetime represents 2019-03-31, weeks is 2 => 2019-04-14
     """
-    pass
- 
+    days_ahead = weeks * 7
+    final_datetime = n_days_ahead(start_datetime, 7*weeks)
+    return final_datetime
+
+def n_days_ahead(start_datetime, days):
+    """
+    Return a datetime that is start_datetime plus days into the future
+    """
+    delta = datetime.timedelta(days = days)
+    return start_datetime + delta
+
+def days_for_this_week(start_datetime):
+    """
+    Return a tuple of two datetimes, one for each lecture day
+    start_datetime is the first lecture day of the week, the second
+    is calculated by start_datetime + 2 days days_ahead
+
+    Warning: Does not account for holidays (see days_without_holidays)
+    """
+
+    day_one = start_datetime
+    day_two = n_days_ahead(start_datetime, 2)
```

Comment:
Seems like this assumes that lectures are 2 days apart. However, a course could be on TWR or any other combination of days. Perhaps, we can have `lecture_days` that is a subset of `legal_days_of_week`? Then, we'd parse that variable to know what offset to use.

Reply:
I added support for any number of days in the newest commit in this pull request.